packages/components/src/Menu/Menu.tsx

```diff
@@ -264,20 +264,34 @@ export interface ActionProps {
    */
   readonly icon?: IconNames;
 
+  /**
+   * Visual style for the action button
+   */
+  readonly buttonVariation?: "default" | "destructive";
+
   /**
    * Callback when an action gets clicked
    */
   onClick?(event: React.MouseEvent<HTMLButtonElement>): void;
 }
 
-function Action({ label, sectionLabel, icon, onClick }: ActionProps) {
+function Action({
+  label,
+  sectionLabel,
+  icon,
+  buttonVariation,
+  onClick,
+}: ActionProps) {
   const actionButtonRef = useRef() as RefObject<HTMLButtonElement>;
+  const buttonClasses = classnames(styles.action, {
```

Comment:
Some of our icons have built in styles, such as trash defaulting to color-destructive (which is why the screenshot looks okay in the PR).

Because of the color defaulting on trash can we also set the [Icon Color manually](https://github.com/GetJobber/atlantis/blob/17a49de3c78e2db92c4f775bc954b362ba8527ce/packages/components/src/DataList/components/DataListAction/DataListAction.tsx#L26) when the `destructive` boolean is present, to ensure they line up in the case an Atlantis consumer chooses an icon other than `destructive`?

Reply:
For sure! Great callout. I'll make that change!